Engine/source/gfx/gfxDrawUtil.h

```diff
@@ -187,6 +187,14 @@ class GFXDrawUtil
    GFXStateBlockRef mRectFillSB;
    
    FontRenderBatcher* mFontRenderBatcher;
+
+   // Expanded shaders
+   GFXShaderRef mRoundRectangleSahder;
+   GFXShaderConstBufferRef mRoundRectangleSahderConsts;
+   GFXShaderConstHandle* mModelViewSC;
+   GFXShaderConstHandle* mRadiusSC;
+   GFXShaderConstHandle* mSizeSc;
+   GFXShaderConstHandle* mRectCenterSc;
```

Comment:
Typos and inconsistencies here. `Sahder` instead of `Shader` and `SC` vs `Sc`.

Reply:
thanks will sort those out